Documentation/Tutorials/BestPractice/ICalendar/Index.rst

```diff
@@ -29,7 +29,7 @@ A very simple way to generate the iCalendar feed is using plain TypoScript. All
 
 .. code-block:: typoscript
 
-    [globalVar = TSFE:type = 9819]
+    [getTSFE() && getTSFE().id == 9819]
```

Comment:
I would suggest to use traverse() also id and type is not the same thing...

Reply:
@linawolf wanna do a PR for that?

Reply:
@linawolf Thank you for the comment, of course you're right. I had taken this from the documentation and overlooked the fact that id is used there instead of type. I will create a new PR in which traverse and type is used.

Reply:
@linawolf Which spelling of `traverse` do you mean exactly?

The notation `[traverse(page, 'type') == 9819]` does not work, because `type` is not a field of the `pages` table and therefore not in the current `page` array, which is used here and contains all record data of the current page.

I see no other way to retrieve the TSFE data or at least the current `type` of the called page than via `[getTSFE() && getTSFE().type == 9819]` (or from version 12 and higher then with the null-safe operator `[getTSFE()?.type == 9819]`).

The same problem has already been discussed here on Forge in [issue #100563](https://forge.typo3.org/issues/100563) with the result of using `getTSFE()` as I originally intended.

In my opinion, this is also the easiest to understand (at least for beginners). In the case of `traverse`, if it comes into question for this use case, you must first understand what this function does.

For these reasons, I will create my new PR with `[getTSFE() && getTSFE().type == 9819]`. Feel free to change this to a `traverse` notation if you know of one and feel this is more appropriate here.